cfme/tests/middleware/test_middleware_server.py

```diff
@@ -4,33 +4,30 @@
 from utils.version import current_version
 
 pytestmark = [
-    pytest.mark.usefixtures('single_middleware_provider'),
+    pytest.mark.usefixtures('setup_provider'),
```

Comment:
then, each test will have impact on other tests, this is potential intermittent failures